app/src/main/java/com/example/nvdovin/weatherapp/presentation/application/dagger/NetworkModule.java

```diff
@@ -0,0 +1,46 @@
+package com.example.nvdovin.weatherapp.presentation.application.dagger;
+
+import com.example.nvdovin.weatherapp.data.network.api.WeatherApi;
+import com.google.gson.Gson;
+
+import javax.inject.Singleton;
+
+import dagger.Module;
+import dagger.Provides;
+import okhttp3.OkHttpClient;
+import okhttp3.logging.HttpLoggingInterceptor;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+@Module(includes = DeserializerModule.class)
```

Comment:
Guys, please pay attention at this line. Max divided NetworkModule and DeserializerModule, but DM used only in NetworkModule. So we need this separation?